ca_gazebo/scripts/publishtagspose.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python  
+import roslib
+import rospy
+import math
+import tf
+import geometry_msgs.msg
+import turtlesim.srv
+
+#Publishes the tag 1 pose relative to the /map frame, to "tags_gts/pose" topic
+if __name__ == '__main__':
+    rospy.init_node('tags_tf_to_pose')
```

Comment:
Call the node something like `tags_tf_2_map_pose`. I think it's more descriptive. I mean, add the map word over there.

Reply:
Adressed